src/js/iov/IOV.js

```diff
@@ -78,13 +94,55 @@ export default class IOV {
       hostname = window.location.hostname;
     }
 
+    // if jwt extract required url parameters.
+    if (jwtUrl === true) {
+
+         //Url: clsp[s]-jwt://<sfs addr>[:9001]/<jwt>?Start=...&End=...
+        var qp_offset = url.indexOf(parser.pathname)+parser.pathname.length
+
+        var i = 0;
+        var qr_args = url.substr(qp_offset).split('?')[1];
```

Comment:
having the ability split on a question mark, is it necessary to get this substring?  also, is it possible for the token to have a question mark in it?  if these points are valid, how about something like this:
```
var qr_args = url.split('?');
qr_args.shift();
qr_args = qr_args.join('?');
```